data/users.json

```diff
@@ -1,7 +1,13 @@
 {
+    " HVision-NKU": {
```

Comment:
Please remove the leading space if you can

Reply:
Do we not do input transformations on user ids? I thought I added something that removes spaced and makes it lowercase.

If not, then we should add that.

Reply:
I'm assuming this was done via the UI, so apparently we don't

Reply:
ah yea my bad, sometimes when copying names I had a whitespace in there somewhere, I remember this happened in the model id input form as well (when adding a new model, I had once trailing whitespaces in the name, then it redirected to a 404 page, when I realized that I had whitespaced in the name and it let to problems that were resolved very quicky tho)